app/assets/javascripts/controllers/cloud_network/cloud_network_form_controller.js

```diff
@@ -63,4 +63,34 @@ ManageIQ.angular.app.controller('cloudNetworkFormController', ['$http', '$scope'
     $scope.angularForm.$setPristine(true);
     miqService.miqFlash("warn", "All changes have been reset");
   };
+
+  $scope.filterNetworkManagerChanged = function(id) {
+    miqService.sparkleOn();
```

Comment:
There are two functions named `filterNetworkManagerChanged()` in this controller, one of them is calling
`getCloudSubnetNetworksByEmsComplete()` which doesn't seem to be present.